src/plugins/data/common/search/aggs/buckets/migrate_include_exclude_format.ts

```diff
@@ -40,6 +45,13 @@ export const migrateIncludeExcludeFormat = {
       if (parsedValue.length) {
         output.params[this.name] = parsedValue;
       }
+    } else if (Array.isArray(value) && value.length > 0 && isStringType(aggConfig)) {
+      // check if is regex
+      const filtered = value.filter((val) => val !== '');
+      if (filtered.length) {
+        const isRegularExpression = isRegex(filtered[0]);
+        output.params[this.name] = isRegularExpression ? filtered[0] : filtered;
+      }
```

Comment:
@flash1293 this is the point I am not sure about. By turning the `include/exclude` to multi means that we are receiving everything as arrays. But the `["ES.*"]` doesn't work, it should be a string to work. So I need a way to identify that this is a pattern and return it as string. For this reason I am using the isRegex function. Does it make sense to you? Better ideas?

Reply:
I see, that's super annoying. Maybe we should discuss tomorrow in the weekly? Not sure how to communicate to the user how this is working, maybe there should be a switch between enumerating values and regex to disambiguate explicitly?